packages/manager/modules/billing/src/components/subscription-tile/subscription-tile.controller.js

```diff
@@ -23,16 +23,13 @@ export default class ServicesActionsCtrl {
   }
 
   $onInit() {
-    this.disableServiceActions = this.$attrs.disableServiceActions
-      ? this.disableServiceActions
-      : false;
+    this.displayServiceActions = this.coreConfig.isRegion(['EU', 'CA']);
```

Comment:
Looks strange that this appears in the `diff`, it doesn't look related to webpaas